src/d3a/d3a_core/sim_results/file_export_endpoints.py

```diff
@@ -64,10 +66,12 @@ def _populate_plots_stats_for_supply_demand_curve(self, area):
                 self.cumulative_offers[area.slug][market.time_slot] = {}
                 self.cumulative_bids[area.slug][market.time_slot] = {}
                 self.clearing[area.slug][market.time_slot] = {}
-            self.cumulative_offers[area.slug][market.time_slot] = market.state.cumulative_offers
-            self.cumulative_bids[area.slug][market.time_slot] = market.state.cumulative_bids
-            self.clearing[area.slug][market.time_slot] = \
-                bid_offer_matcher.matcher.match_algorithm.state.clearing
+            self.cumulative_offers[area.slug][market.time_slot] = {
+                time: value["cumulative_offers"] for time, value in market.state.items()}
```

Comment:
Doesn't this work ?
```suggestion
                time: value["cumulative_offers"] for time, value in bid_offer_matcher.match_algorithm.state.items()}
```

Reply:
unfortunately, it wouldn't work as ATM bid_offer_matcher.match_algorithm.state != {time: cumulative_offers}

Reply:
Give this commit a look: https://github.com/gridsingularity/d3a/pull/1195/commits/69654b167e5639d3893338ddfd5eb8689229b1e6